runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9806,6 +9806,36 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL(node->getNumChildren() == 3, "Wrong number of children in inlineFPTrg1Src3");
+
+   TR::DataType type = node->getDataType();
+   TR_ASSERT_FATAL(type == TR::Float || type == TR::Double, "Wrong node DataType in inlineFPTrg1Src3");
```

Comment:
Again can this be more helpful in telling you what went wrong?

Reply:
Fixed in 2f9a77